<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgImportCSV_Step2</class>
 <widget class="QDialog" name="DlgImportCSV_Step2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSV Importer, Step2</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Double click on a cell to edit the item:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,1">
       <item>
        <widget class="CSVDataTableWidget" name="tab">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Hint:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="laMsg">
           <property name="text">
            <string>--</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="laCount">
       <property name="text">
        <string>Warnings</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnDelete">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>239</red>
               <green>41</green>
               <blue>41</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>239</red>
               <green>41</green>
               <blue>41</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>190</red>
               <green>190</green>
               <blue>190</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Delete row</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnInsert">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>78</red>
               <green>154</green>
               <blue>6</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>78</red>
               <green>154</green>
               <blue>6</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>190</red>
               <green>190</green>
               <blue>190</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Insert row</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnImport">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSVDataTableWidget</class>
   <extends>QTableWidget</extends>
   <header>ui/DlgImportCSV_Step2.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgImportCSV_Step2</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>840</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tab</sender>
   <signal>currentCellChanged(int,int,int,int)</signal>
   <receiver>DlgImportCSV_Step2</receiver>
   <slot>onCellSelectionChanged(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnImport</sender>
   <signal>clicked()</signal>
   <receiver>DlgImportCSV_Step2</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>926</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCellSelectionChanged(int,int)</slot>
 </slots>
</ui>
